### ISSUE TITLE
How to conduct pretraining tasks

### ISSUE THREAD

**Issue by chenyuaner** [hspark1212/MOFTransformer#166]
I want  to pre-train with a million hypothetical MOFs How do i do that?

**Comment by Yeonghun1675** [comment]
Hi @chenyuaner,

If you want pretrain, you can set the `loss_names` to [vfp, moc, mtp]. If you want to add a new pre-train task, you need to modify the code, or use `regression` or `classification` loss instread of it.

**Comment by chenyuaner** [comment]
> Hi [@chenyuaner](https://github.com/chenyuaner),
> 
> If you want pretrain, you can set the `loss_names` to [vfp, moc, mtp]. If you want to add a new pre-train task, you need to modify the code, or use `regression` or `classification` loss instread of it.

I also want the json files of vfp、mtp and moc, could you send me a copy? This is my email: [csy3311582@163.com](mailto:csy3311582@163.com)

**Comment by Yeonghun1675** [comment]
Hi @chenyuaner,

You will receive an email with a zip file of your JSON files. Good luck with your project!

**Comment by chenyuaner** [comment]
> Hi [@chenyuaner](https://github.com/chenyuaner),
> 
> You will receive an email with a zip file of your JSON files. Good luck with your project!

Thanks very much !
